### ISSUE TITLE
Inconsistent Warning For Named Slot in "is" Components

### ISSUE THREAD

**Issue by pxwee5** [vuejs/vue#10168]
### Version
2.6.10

### Reproduction link
[https://codepen.io/jameswee/pen/gNwZxJ?editors=1010](https://codepen.io/jameswee/pen/gNwZxJ?editors=1010)

### Steps to reproduce
In the pen provided above, you can't see the warning because there's no build step. 

However if you put this through Webpack you'll see this error
`<template v-slot> can only appear at the root level inside the receiving the component`

### What is expected?
You will see `<template v-slot> can only appear at the root level inside the receiving the component`

### What is actually happening?
I'm seeking explanation here as well. LOL

---
## Additional Comments

There's an inconsistency in the old and new slot syntax.

The new syntax caused warning when used with the special `is` attribute. 
Both the HTML below are server rendered by a CMS and is compiled by Vue compiler.
```html
<table>
  <tr is="my-tr">
    <template v-slot:default>
      <td>Test</td>
    </template>

    <template v-slot:named>
      <td>Test</td>
    </template>
  </tr>
</table>
```
Error message:
```
<template v-slot> can only appear at the root level inside the receiving the component

217|  <table>
218|    <tbody><tr is="my-tr">
219|      <template v-slot:default="">
   |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
220|        <td>Test</td>
   |  ^^^^^^^^^^^^^^^^^^^
221|      </template>
   |  ^^^^^^^^^^^^^^^
```

The old syntax works without error. 👍
```html
<table>
  <tr is="my-tr">
    <template slot="default">
      <td>Test</td>
    </template>

    <template slot="named">
      <td>Test</td>
    </template>
  </tr>
</table>
```
Can we treat `is` attributed DOM element as vue components like how the old slot syntax works?

I'd like to point out that the default slot works without errors. But ideally we should be able to use named slots here as well.
```html
<table>
  <tr is="my-tr">
    <td>Test</td>
  </tr>
</table>
```

**Comment by LinusBorg** [comment]
use `vue.js` instead of `vue.min.js`

The minified version is meant for production and doesn't contain warnings. This is documented.

**Comment by zrh122** [comment]
@LinusBorg , this may be same as PR #10152.

**Comment by pxwee5** [comment]
@LinusBorg Thanks for pointing that out. But that isn't the fix.

Thanks for pointing to the related issue @zrh122 that's exactly what I'm dealing with.

**Comment by LinusBorg** [comment]
Oh, I see. I thought the issue was about the error message not appearing in the codepen. Thanks for the correction.

In that case: closed as duplicte of #10152

**Comment by LinusBorg** [comment]
Oh, I see. I thought the issue was about the error message not appearing in the codepen. Thanks for the correction.

In that case: closed as duplicte of #10152

**Comment by luckydonald** [comment]
Using `<template slot="named">` instead of `<template v-slot:named>` fixes the issue.
Thanks.
